lib/DropdownMenu/styles.scss

```diff
@@ -57,7 +57,7 @@ $dropdown-menu-link-padding: 7px 15px !default;
 }
 
 /** Custom **/
-.dropdown__link {
+.dropdown__link, .dropdown-menu a.dropdown__link {
```

Comment:
Do we need this declaration to override some styling?

Reply:
If we don't the styles get overridden by .dropdown-menu > li > a. Im open to other ways around it though